viewer/CMakeLists.txt

```diff
@@ -3,6 +3,11 @@ find_package(SDL)
 if (SDL_FOUND)
 	include_directories(${SDL_INCLUDE_DIR})
 	set(EXTRA_LIBS ${SDL_LIBRARY} ${EXTRA_LIBS})
+	if (APPLE)
+		execute_process(COMMAND "sdl2-config" "--static-libs" OUTPUT_VARIABLE SDL2_EXTRAS)
```

Comment:
Not necessary if FindSDL.cmake can find the dylib, but for some reason I am having trouble with this. **You can remove this clause**, but you might have to tell CMake where to find SDL. The following works with the latest homebrew packages:
```
cmake enki \
  -DSDL_LIBRARY=/usr/local/Cellar/sdl2/2.0.4/lib/libSDL2.dylib \
  -DSDL_INCLUDE_DIR=/usr/local/Cellar/sdl2/2.0.4/include/SDL2
```

By the way the `--static-libs` flag to `sdl2-config` adds the system frameworks, -framework CoreVideo -Wl -framework Cocoa -Wl -framework Carbon -Wl -framework IOKit -Wl -framework CoreAudio -Wl -framework AudioToolbox -Wl -framework AudioUnit